dipy/workflows/reconst.py

```diff
@@ -120,9 +121,7 @@ def run(self, input_files, bvalues, bvectors, mask_files, b0_threshold=0.0,
             data = img.get_data()
             affine = img.affine
 
-            if mask is None:
-                mask = None
-            else:
+            if mask is not None:
```

Comment:
OK - I am not sure why this is here. @Garyfallidis : under what circumstances would the mask be a `None`? It's a required input into the workflow?

Reply:
When you run the Flow from a Python script it is easy to put a None.  But not easy from the command line. It may be good to enforce the same behavior for consistency.

Reply:
That makes sense! What do you think about defaulting to `None`, so that if users don't pass a mask, we use this option? We can do that on a separate PR, to make sure to do this uniformly across the different flows.

Reply:
All that's to say: I think this PR is ready to go! @skoudoro : would you mind giving this another look?

Reply:
I think it will be good to just raise an exception if there is no mask. In that way, 
1. we keep the consistency with the command line
2. we give good habits to the user by forcing him to generate a mask

But I agree with @arokem, we can do that in a separate PR